x-pack/plugins/secops/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "react-beautiful-dnd": "^10.0.1",
+    "react-markdown": "^4.0.6",
```

Comment:
`react-markdown` is already a dependency of Kibana, this just bumps the version to latest (in the plugin)